src/Card/SummaryCard/SummaryCard.stories.tsx

```diff
@@ -232,11 +230,12 @@ export const ScenarioExample = {
       </TableContainer>
     ),
     labels,
-    media: "https://picsum.photos/400/200",
+    media: "https://picsum.photos/336/190",
```

Comment:
Please add a story for verison with infographic.

Reply:
Line 238

Reply:
I saw it in the Scenario Example, what i mean is a seperate story for WithVersionChip